docs/concepts.rst

```diff
@@ -333,10 +333,7 @@ won’t detect changes of the following types:
    global variables that have simple types (int, string, bool, bytes, etc.), but
    it will ignore changes for other types of variables.
 
-3. **Python Classes**: Bionic won’t detect changes to the code of Python classes.
```

Comment:
Hmmm. Offline I agreed that we don't need to generate warnings for things we don't handle yet, like `attrs` attributes. But if we're going to say in the document that we hash classes, then it does seem appropriate to emit warnings for stuff we don't support yet. Maybe we should say something like "we detect some but not all changes" here.